### PR TITLE
redirect

### PR DESCRIPTION
**Pull request by coati-00**
Currently navigating directly to https://training.lowernysphtc.org/ returns a "Bad Request (400)"

### CODE REVIEW

File: phtc/main/tests/test_views.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def setUp(self):
 
     def test_index(self):
         result = self.c.get("/")
-        self.assertEqual(result.status_code, 200)
+        self.assertEqual(result.status_code, 301)
```
Comment (thraxil) [inline phtc/main/tests/test_views.py]: I'd recommend against using 301 redirects within the app. A 301 is permanent and browsers cache them forever, so they can really never be undone.

Comment (coati-00) [inline phtc/main/tests/test_views.py]: So what return code should I use?

**Comment by sdreher** [comment]
Just a note:  https://training.lowernysphtc.org/ is likely failing as ssl is not (and has never been afaik) configured for this site. Direct navigation to http://training.lowernysphtc.org works fine.

File: phtc/main/tests/test_views.py (hunk 1 above)
Comment (nikolas) [inline phtc/main/tests/test_views.py]: It looks like 303 and 307 can both be used as temporary redirects: https://en.wikipedia.org/wiki/URL_redirection#HTTP_status_codes_3xx

**Comment by thraxil** [comment]
What is the status of this PR?

**Comment by thraxil** [comment]
I'm just going to close this out now then...

**Pull request closed without merging**